darwin/options.py

```diff
@@ -209,6 +208,28 @@ def __init__(self):
             help="Confirmation flag to delete the file without prompting for manual input.",
         )
 
+        # Add comments
+        parser_comment = dataset_action.add_parser("comment", help="Comment image.")
+        parser_comment.add_argument(
+            "dataset",
+            type=str,
+            help="[Remote] Dataset name: to list all the existing dataset, run 'darwin dataset remote'. ",
+        )
+        parser_comment.add_argument("file", type=str, help="File to comment")
+        parser_comment.add_argument("--comment-text", type=str, help="Comment: list of words")
+        parser_comment.add_argument(
+            "--comment-x-position", required=False, type=float, default=1, help="X coordinate for comment"
+        )
+        parser_comment.add_argument(
+            "--comment-y-position", required=False, type=float, default=1, help="Y coordinate for comment"
+        )
+        parser_comment.add_argument(
+            "--comment-width-px", required=False, type=float, default=1, help="Comment width in pixels"
+        )
+        parser_comment.add_argument(
+            "--comment-height-px", required=False, type=float, default=1, help="Comment height in pixels"
+        )
+
```

Comment:
What do you think of simplifying these arg names? After all, we already use `comment` as a command:

```suggestion
        parser_comment.add_argument("--text", type=str, help="Comment string")
        parser_comment.add_argument(
            "-x", type=float, default=1, help="X coordinate for comment"
        )
        parser_comment.add_argument(
            "-y", type=float, default=1, help="Y coordinate for comment"
        )
        parser_comment.add_argument(
            "-w", "--width", type=float, default=1, help="Comment width in pixels"
        )
        parser_comment.add_argument(
            "-h", "--height", type=float, default=1, help="Comment height in pixels"
        )

```

I also get why we'd like to keep `default` values set to `1`, but we should actually check that doing stuff like this works and the backend validates it correctly
```
darwin dataset comment --text "Will be placed at (1, 1), width 1px, height 1px"
```

Reply:
I am OK with simplifying names, but I am not sure what you mean by "works". I would not have asked for a review if I had not tested it :P

Reply:
Simplified args as requested.

Reply:
> I would not have asked for a review if I had not tested it

Of course I know and trust that, but some things may just slip through, just wanted to be 100% sure

Reply:
:+1: